src/service.ts

```
@@ -690,5 +716,400 @@ export class KnowledgeService extends Service {
+          agentId: this.runtime.agentId,
+          documents: documents.map(doc => ({
+            id: doc.id,
+            content: doc.content,
+            metadata: options.includeMetadata ? doc.metadata : undefined,
+            createdAt: doc.createdAt,
+          })),
+        }, null, 2);
+
+      case 'csv':
+        // Simple CSV export
+        const headers = ['ID', 'Title', 'Content', 'Type', 'Created'];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)